### DISCUSSION TITLE
How to recover from a deleted `public/system/cache`?

### DISCUSSION THREAD

**Discussion by ISibboI** [mastodon/mastodon#21305] [Q&A] [answered]
I tried googling, but nothing comes up.

I recently run out of disk space on my server because my `public/system/cache` directory became too large. I have then decided that since it is called "cache" it should be safe to delete. But apparently it was not. Many profiles I follow from foreign servers now miss their profile picture and banner.

How can I make mastodon recover from this? Is there a way to redownload all missing profile pictures automatically?

**Comment by gianarb** [comment]
I did the same mistake, not sure how to fix it, but yep, cache sounds way more important that what the name suggests :smile:

**Reply by gianarb** [reply, marked as answer]
Ok this is how I got a lot of what I have lost back:

```
tootctl accounts refresh --all
```

**Reply by ISibboI** [reply]
Great, that seems to work well! I will still leave this open, hoping that someone would give an "official" answer.

**Reply by brindledcat** [reply]
Thank you! I've spent hours trying to figure this out :)

**Reply by chatnoirslay** [reply]
> Great, that seems to work well! I will still leave this open, hoping that someone would give an "official" answer.

Its the official tootctl (mastodon command line) so I suppose you should mark it as closed.
